layers/+completion/ivy/funcs.el

```diff
@@ -254,27 +254,30 @@ that directory."
                    (buffer-substring-no-properties
                     (region-beginning) (region-end))
                  (thing-at-point 'symbol t))))
-    (counsel-git-grep nil input)))
+    (counsel-git-grep input)))
```

Comment:
I'm not sure whether this function is useful at all. Thing at point can be better guessed by running `counsel-git-grep` followed by `M-n`.

Reply:
`M-n` which calls `ivy-previous-history-element`, only cycles backwards through the prompts history. If one hasn't manually typed in the symbol under the point then it won't be listed.

This PR fixes the issue where `SPC s g P` behaved the same as `SPC s g p` and didn't insert the symbol under the point.

Reply:
@duianto `M-n` calls the next (not previous) element in history. When the next element does not exist, it tries to predict what the user wants to input, in this case, what the thing at point is. See https://oremacs.com/2016/02/15/counsel-ffap/

Reply:
Your right, `M-n` does pick up the symbol under the cursor.
I did not know that. Thanks for the explanation.

I'm used to navigating backwards through the history at prompts, therefore I assumed that you meant `M-p`, but I still managed to type the correct key binding: `M-n` 😄 

Still, `SPC s g P` was broken, but it's fixed now.